docs/TypeConstructor.md

```diff
@@ -9,8 +9,10 @@ value.
 A constructor function can be provided that is called to convert incoming values set on this property, like:
 
 ```js
-prop: {
-  Type: Person
+{
```

Comment:
I prefer these examples without the surrounding `{}` because it shows there might be other properties.  But this isn't important.

Reply:
It was necessary for the parsers that were updating for let/const and for the linter.